Source/PlasticSourceControl/Private/SPlasticSourceControlBranchesWidget.cpp

```diff
@@ -553,38 +562,59 @@ TSharedPtr<SWidget> SPlasticSourceControlBranchesWidget::OnOpenContextMenu()
 
 	FToolMenuSection& Section = *Menu->FindSection("Source Control");
 
+	static const FText CreateChildBranchTooltip(LOCTEXT("CreateChildBranchTooltip", "Create child branch from the selected branch."));
+	const FText& CreateChildBranchTooltipDynamic = bSingleSelection ? CreateChildBranchTooltip : SelectASingleBranchTooltip;
 	Section.AddMenuEntry(
 		TEXT("CreateChildBranch"),
 		LOCTEXT("CreateChildBranch", "Create child branch..."),
-		LOCTEXT("CreateChildBranchTooltip", "Create child branch from the selected branch."),
+		CreateChildBranchTooltipDynamic,
 		FSlateIcon(),
 		FUIAction(
 			FExecuteAction::CreateSP(this, &SPlasticSourceControlBranchesWidget::OnCreateBranchClicked, SelectedBranch),
-			FCanExecuteAction::CreateLambda([this, SelectedBranch]() { return (!SelectedBranch.IsEmpty()); })
+			FCanExecuteAction::CreateLambda([bSingleSelection]() { return bSingleSelection; })
 		)
 	);
 
+	static const FText SwitchToBranchTooltip(LOCTEXT("SwitchToBranchTooltip", "Switch the current workspace to this branch."));
+	const FText& SwitchToBranchTooltipDynamic = bSingleNotCurrent ? SwitchToBranchTooltip : bSingleSelection ? SelectADifferentBranchTooltip : SelectASingleBranchTooltip;

```

Comment:
The one-liners are a bit difficult to read, perhaps you could refactor them into static methods? Or at least indent them in a more visual way 🙏🏻

Reply:
I made a change to have only one test per line, and align them all with same indentation, so this one now reads better in 3 lines